Makefile

```diff
@@ -4,6 +4,17 @@ VERSION_MOCKGEN=v1.6.0
 RELEASE_NAME_DB=db
 RELEASE_NAME_APP=app
 
+# helm chart path
+HELM_CHART_DB = helm-charts/database-to-integrity-sum
+HELM_CHART_APP = helm-charts/app-to-monitor
+
+# DB params
+DB_NAME	?= $(PGDATABASE)
+DB_USER	?= $(PGUSER)
+DB_PASSWORD	?= $(PGPASSWORD)
```

Comment:
Setting `PGPASSWORD` and other DB envs,  you can just export `DB_PASSWORD`

Reply:
done